usecases/usecases.go

```diff
@@ -82,7 +72,7 @@ func (usecases *Usecases) NewExportScheduleExecution() *scheduledexecution.Expor
 
 func (usecases *Usecases) NewPopulateOrganizationSchema() organization.PopulateOrganizationSchema {
 	return organization.PopulateOrganizationSchema{
-		TransactionFactory:           &usecases.Repositories.TransactionFactoryPosgresql_deprec,
+		ClientSchemaExecutorFactory:  usecases.NewClientDbExecutorFactory(),
```

Comment:
I notice we move from `NewOrgTransactionFactory` to `NewClientDbExecutorFactory` but we still have `NewTransactionFactory`: is there a reason to distinguish the naming clearly like this ?

Reply:
NewTransactionFactory is now for generating transactions (on both schema types) while NewClientDbExecutorFactory creates an executor (wrapping a pool)

Reply:
(and it's the same underlying implementation, but separate interfaces)